common/src/main/java/io/druid/metadata/MetadataStorageActionHandler.java

```diff
@@ -104,9 +104,8 @@ public void insert(
    * Remove the lock with the given lock id.
    *
    * @param lockId lock id
-   * @return true if the lock was removed, false if the given lock id did not exist
    */
-  public boolean removeLock(long lockId);
+  public void removeLock(long lockId);
```

Comment:
It does already verify that; if the method doesn't throw an exception, then we're guaranteed that no lock exists with id = lockId (either it was deleted or it never existed).